src/Proj4.jl

```diff
@@ -34,7 +34,7 @@ end
 
 "Get human readable error string from proj.4 error code"
 function _strerrno(code::Cint)
-    bytestring(ccall((:pj_strerrno, libproj), Cstring, (Cint,), code))
+    bytestring(ccall((:pj_strerrno, libproj), Compat.Cstring, (Cint,), code))
```

Comment:
Almost!  This won't work on julia 0.4 though :-/  I think you simply want to use `Cstring` with no decoration - `using Compat` should bring the declaration into scope correctly.